.gitignore

```diff
@@ -23,12 +23,9 @@ cudf.egg-info/
 python/build
 python/*/build
 python/cudf/cudf-coverage.xml
-python/cudf/*/_lib/**/*.cpp
+python/cudf/*/_lib/**/*\.cpp
```

Comment:
Is this `\` intended?

Reply:
Yes -- good catch! My intent here is to *not* ignore the `cudf/_lib/cpp` directory, but to still ignore all files in `cudf/_lib` with the `.cpp` extension.

Do you think this will achieve that?

Reply:
I believe it will.